pom.xml

```diff
@@ -406,6 +407,11 @@
             </dependency>
 
             <!-- Debezium artifacts -->
+            <dependency>
```

Comment:
Do we need to do anything to add the new module to assemblies?

Reply:
@gunnarmorling No, I just double checked and build the plugin packages and they contain new API module